pkg/gardenlet/operation/botanist/vpa.go

```diff
@@ -83,7 +83,6 @@ func (b *Botanist) DefaultVerticalPodAutoscaler() (vpa.Interface, error) {
 		b.SecretsManager,
 		vpa.Values{
 			ClusterType:              component.ClusterTypeShoot,
-			Enabled:                  true,
```

Comment:
Don't we have to destroy the component here when VPA is disabled?

Reply:
Already handled separately in https://github.com/gardener/gardener/blob/841e737a8bcab1626a7995c5798a4cbeb1f72156/pkg/gardenlet/operation/botanist/vpa.go#L97-L103